routes/feedback.py

```diff
@@ -1,32 +1,36 @@
-
 from config.db import get_database
 from fastapi import APIRouter, Response
-
 from models import events as eventModel
-
-#from docs import feedback as docs
+# from docs import feedback as docs
 import logging
 import util.feedback as utils
+from util import events as utilss
+
 from starlette.exceptions import HTTPException
 
 router = APIRouter()
 
+
 @router.delete(
     "/feedback/delete/{event_id}/{feedback_id}",
-    #description=
-    #summary=
-    #tags=
+    # description=
+    # summary=
+    # tags=
     status_code=204,
 )
-async def delete_event(event_id, feedback_id):                   
-
+async def delete_event(event_id, feedback_id):
     # get DB Database
     db = get_database()
 
     # perform deletion
     await utils.delete_feedback(event_id, feedback_id, db)
 
     # prevent console error
-    return Response(status_code=204) # https://github.com/tiangolo/fastapi/issues/717#issuecomment-583826657
-   
-    
\ No newline at end of file
+    return Response(status_code=204)  # https://github.com/tiangolo/fastapi/issues/717#issuecomment-583826657
+
+
+@router.get("/users/{feedback}", response_model=eventModel.Events, status_code=201)
```

Comment:
This method is missing a couple of things and has a couple of errors:

- the `description` and `summary` are missing (commented out code or a single `# TODO` would be fine in the meanwhile)

	- the route that the client is going to use is going the main parameter of the function, so instead of `"/users/{feedback}"` it should be something like `"/feedback/find/{feedback_id}"`.

	- there's a couple of things to unpack with the renaming:

	- routes with the same leading word are grouped together (e.g. `"/users/find"` gets grouped with `"/users/register"`)

	- the `feedback` query parameter doesn't really tell us anything about the input; `feedback_id` is much clearer

	- the response model is `eventModel.Events` meaning that this endpoint is expected to return an event object (instead of a feedback object); `eventModel.Events` -> `models.get_feedback_response`
	- `models.get_feedback_response` would look something like:  (assuming that the `Feedback` class is the main instance representation of a feedback object)
	```python
	class get_feedback_response(Feedback):
		pass
	```